resources/assets/logisticspipes/book/en_us/book.json

```diff
@@ -1,4 +1,72 @@
 {
-  "title": "Logistics Pipes Quickstart Guide",
-  "page_count": 10
+  "title": "Logistics Pipes Compendium",
+  "divisions": [
+    {
+      "title": "Guides",
+      "chapters": [
+        {
+          "title": "Quickstart Guide",
+          "item": "minecraft:apple",
+          "pages": "10"
```

Comment:
Instead of "title" and "pages" in the main file, put them in the directory with all the pages, so a chapter is self-contained.
And of course, add a path to the chapter here instead.

Reply:
why is pages a String?

Reply:
So having a .json file for the book and one for each chapter? Sounds good but I guess I need to have everything that is not working fixed first :/

Reply:
Added that to my Glo board

Reply:
The page is a string for no reason to be honest when calling into the Object it gets turned into an int

Reply:
Why is this (and others) marked as resolved if it, well, isn't? When you fix it with a commit, it autocollapses, until then, keep comments open so that you can actually see what is resolved and what isn't, please.